capable/types/src/lib.rs

```diff
@@ -44,30 +45,30 @@ impl From<SgxError> for Error {
 }
 
 /// Try to convert an
-/// [sgx_device_status_t](mc_sgx_capable_sys_types::sgx_device_status_t) to an
-/// [Error].
+/// [`sgx_device_status_t`](mc_sgx_capable_sys_types::sgx_device_status_t) to an
+/// [`Error`].
 ///
 /// This is fallible because device_status_t also includes
 /// [`SGX_ENABLED`](mc_sgx_capable_sys_types::sgx_device_status_t::SGX_ENABLED),
 /// which is (obviously) not an error.
```

Comment:
:wrench: Looks like I missed this in the initial review consider omitting "obviously", it falls into the potentially inflammatory words, https://www.knowledgeowl.com/home/dont-say-simply-jim-fisher
```suggestion
/// which is not an error.
```